### ISSUE TITLE
Statement Position Error: If there is a word including "catch" after the line break.

### ISSUE THREAD

**Issue by hiseri** [realm/SwiftLint#237]
Please see an example below.
```Swift
struct SwiftLint {
    let title: String
    let catchphrase: String
}

let hoge = SwiftLint(
    title: "title",
    catchphrase: "catchphrase" // Warning Occurred
)
```
Warning is ...
```
Statement Position Violation: Else and catch should be on the same line, one space after the previous declaration. (statement_position)
```

Check this issue, please.
Thank you.

**Comment by jpsim** [comment]
Thanks for reporting this along with a good repro case. This is now fixed in #238.

**Comment by dawson86** [comment]
add space after the statement
